quickstart/faqs.md

```diff
@@ -87,4 +87,9 @@ In Kubernetes, when a PVC is created with the reclaim policy set to 'Retain', th
 
   ### How does the PV garbage collector work?
 
-   The PV garbage collector deploys a watcher component, which subscribes to the Kubernetes Persistent Volume deletion events. When a PV is deleted, an event is generated by the Kubernetes API server and is received by this component. Upon a successful validation of this event, the garbage collector deletes the corresponding Mayastor volume resources.
\ No newline at end of file
+   The PV garbage collector deploys a watcher component, which subscribes to the Kubernetes Persistent Volume deletion events. When a PV is deleted, an event is generated by the Kubernetes API server and is received by this component. Upon a successful validation of this event, the garbage collector deletes the corresponding Mayastor volume resources.
+
+
+## How to disable cow for btrfs filesystem?
+
+To disbale cow for `btrfs` filesystem, use `nodatacow` as a mountOption in the storage class which would be used to provision the volume.
```

Comment:
Do we need to document this at all? @Abhinandan-Purkait

Reply:
No this is not mandatory, the only reason I asked Archita to place was because system tests use this way. We can totally skip this. @tiagolobocastro  @avishnu  What do you think is it worth putting this?